src/index.html

```diff
@@ -6,9 +6,63 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./reset.css">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="wrapper">
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav>
+        <ul class="tab-container">
+          <li class="tab">
+            <a class="nav-link is-active" href="#">
+              apple
+            </a>
+          </li>
+          <li class="tab">
```

Comment:
```suggestion
          <li class="tab">
            <a class="nav-link is-active" href="#">
              apple
            </a>
          </li>
          
          <li class="tab">
```
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/21f1c789-daa7-43ab-a34c-387b3147bb36)